src/App.vue

```diff
@@ -823,6 +833,23 @@ export default {
         this.nameRequestInvalidDialog = false
         this.fetchData()
       }
+    },
+
+    /** Request and Download Business Summary Document. */
+    async downloadBusinessSummary (): Promise<void> {
+      this.showSpinner = true
+      const summaryDocument: DocumentIF = {
+        title: 'Summary',
+        filename: `${this.businessId} - Summary - ${this.getCurrentDate}.pdf`,
+        link: `businesses/${this.businessId}/documents/summary`
```

Comment:
With the introduction of the summary document, a new endpoint was created `businesses/${this.businessId}/documents` which exists in parallel with our filings documents endpoint.
ie `businesses/${this.businessId}/filings/${someFilingId}/documents`. 

The response of the new endpoint returns `{ summary: 'url' }` where the url is essentially the same url as the new one listed above but with `/summary` tacked onto the end, so i opted to just make this method specific and retrieve the summary document directly. 

IF in the future they decide to include more document/document types on the business documents endpoint, we would need 2 requests. I opted not to make 2 requests for a single document in this instance.

Reply:
To elaborate, the 1 request would be to retrieve all the document links. As it stands right now, it's a single object containing the `summary` key, with value `url/summary`.

Reply:
I'm OK with this.

Reply:
> > SonarCloud is unhappy with code duplication on 4 files i did not touch.. so not sure 🤷
> 
> See https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/10572

ahhh ok, so this isn't the first time. Thank you for sharing!